src/collective/cover/tests/test_base_tile.py

```diff
@@ -66,24 +65,3 @@ def test_default_configuration(self):
 
     def test_accepted_content_types(self):
         self.assertEqual(self.tile.accepted_ct(), ALL_CONTENT_TYPES)
-
-    def test_delete_tile_persistent_data(self):
```

Comment:
I would like to keep this test to guarantee all persistent data, conffiguration an permissions are removed at low level and the `ObjectModifiedEvent` is being fired.

Reply:
This test uses the `tile.delete()` method that was suggested to be removed.

Reply:
we have to think on how to test the feature on low level.